tendrl/node_agent/manager/__init__.py

```diff
@@ -102,6 +104,11 @@ def main():
 
     complete = threading.Event()
 
+    if "tendrl/monitor" in NS.node_context.tags:
```

Comment:
You'll need to check

NS.tendrl_context.sds_name == "gluster"

before starting the gluster sync thread

Reply:
@r0h4n Do you want this sync thread to be run on all the gluster nodes ?
I felt its enough if we run this thread only on monitor node

Reply:
If we have all the required details on the monitor node, it looks fine to me to run this on monitor node only.

Reply:
ok, then we need check if the gluster_integration_sync thread has started and if not then start it.
We need to move this logic to https://github.com/Tendrl/node-agent/blob/master/tendrl/node_agent/node_sync/__init__.py#L100

Reply:
Could you pls provide the reasoning behind this suggestion ? I see the ceph sync thread is started here: https://github.com/Tendrl/node-agent/blob/master/tendrl/node_agent/manager/__init__.py#L101

Also what is the point of checking if thread is running on every node sync ?

Reply:
- You'll not see the "tendrl/monitor" tag before the node_sync. So this code is not even going to run if you put it in manager.

- The reason to check if the thread is started/running in the node_sync is because you dont want to start a new instance of gluster_integration_thread every sync

- The ceph_integration thread start code above also needs to be moved to the node_sync, but not sure if it belongs on the monitor because it tries to do integration_id specific things. Id prefer if you remove this code all together, we dont support ceph anymore.

Reply:
ack